package.json

```diff
@@ -636,6 +636,11 @@
             {
                 "type": "docker",
                 "label": "Docker: Debug in Container",
+                "languages": [
+                    "csharp",
+                    "razor",
+                    "aspnetcorerazor"
+                ],
```

Comment:
should we add python & node.js here too? so when you're on a .py file or .js file and presses `f5`, you'll be able to see our debugger. I was thinking we can do that in a separate PR.

Reply:
I think another PR is fine